tests/lib/rules/prefer-includes.js

```diff
@@ -0,0 +1,36 @@
+const { RuleTester } = require('eslint/lib/rule-tester')
+const rule = require('../../../lib/rules/prefer-includes');
+
+const ruleTester = new RuleTester();
+
+ruleTester.run('prefer-includes', rule, {
+    valid: [
```

Comment:
My only comment here would be to include a call to parseInt such that it used to trigger the error - I want to say it's a parseInt of a structured element ("parseInt(foo.bar,10)") but I'd have to test this further to validate.